Cargo.toml

```
@@ -26,5 +26,5 @@ objc_id = "0.1"
 objc-foundation = "0.1"
 
 [target.'cfg(all(unix, not(any(target_os="macos", target_os="android", target_os="ios", target_os="emscripten"))))'.dependencies]
-x11-clipboard = { version = "0.5.1", optional = true }
+x11-clipboard = { version = "0.6", optional = true }
```

```suggestion
x11-clipboard = { version = "0.6.1", optional = true }
```

Should **always** specify the minimum semver version actually tested.

---

sounds reasonable, just updated. thank you